.bazelrc

```diff
@@ -62,3 +62,6 @@ build:ubsan --copt -g
 build:ubsan --strip=never
 build:ubsan --copt -fno-omit-frame-pointer
 build:ubsan --linkopt -fsanitize=undefined
+
+# Import site-specific configuration.
+try-import .bazelrc-site
```

Comment:
Moved the try-import line to end of file.  Though it is unlikely that there would be a conflict in configurations, this would give priority to `.bazelrc-path`. What do people think?